app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Rails BlogApp</h1>
\ No newline at end of file
+<ul class="users-list">
+  <% @users.each do |user| %>
+    <li>
+      <ul>
+        <li>
+          <img src="<%= user.photo %>" alt="user_pic" />
+        </li>
+        <li>
+          <%= user.name %>
+          <p class="posts_count">Number of posts: <%= user.posts_counter || 0 %></p>
+        </li>
+      </ul>
+    </li>
+  <% end %>
+</ul>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169435699-b944a1dd-1409-4342-a2a3-327386fb355c.png)


- [ ] Kindly note that you need to follow the wireframes that were provided in the requirements, I know that the style does not matter but you need to add some basic CSS to follow the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md). check the wireframes :point_down: 

![image](https://user-images.githubusercontent.com/65068771/169435586-73bdb804-e322-44fe-a337-6f7ca8c310d0.png)

- [ ] Kindly also make sure when I click on the user it redirects me to the `user#show` view page

Reply:
Addressed